systems/registry/node/pkg/server/node.go

```diff
@@ -228,6 +237,32 @@ func (n *NodeServer) GetNodesByState(ctx context.Context, req *pb.GetNodesByStat
 	return resp, nil
 }
 
+func (n *NodeServer) List(ctx context.Context, req *pb.ListRequest) (*pb.ListResponse, error) {
+	log.Infof("List nodes by nodeId: %v, siteId: %v, networkId: %v, connectivity: %v, state: %v, type: %v", req.GetNodeId(), req.GetSiteId(), req.GetNetworkId(), req.GetConnectivity().String(), req.GetState().String(), req.GetType())
+
+	var connectivity, state *uint8
+	if req.GetConnectivity().String() != Undefined {
```

Comment:
Why are we using **undefined** constant when we already have ``ukama.NodeConnectivityUndefined` ? `